integreat_cms/api/urls.py

```diff
@@ -123,6 +124,16 @@
 urlpatterns = [
     path("api/regions/", include(region_api_urlpatterns)),
     path("wp-json/extensions/v3/sites/", include(region_api_urlpatterns)),
+    path(
+        "api/social/<slug:region_slug>/<slug:language_slug>/",
+        include(
+            [
+                re_path(
+                    r"^(?P<path>\w+)$", socialmedia_headers, name="socialmedia_headers"
```

Comment:
```suggestion
                path(
                    "<path:path>/", socialmedia_headers, name="socialmedia_headers"
```
For me, this regex didn't work for non-root pages. However, we don't even need the regex since Django supports paths per default via the [`path` converter](https://docs.djangoproject.com/en/3.2/topics/http/urls/#path-converters).